test/es-module/test-esm-import-meta-main-eval.mjs

```diff
@@ -0,0 +1,42 @@
+import { spawnPromisified } from '../common/index.mjs';
+import * as fixtures from '../common/fixtures.js';
+import assert from 'node:assert/strict';
+
+const execOptions = ['--input-type', 'module'];
```

Comment:
nit: I find it a bit over the top to use an array here, it'd be probably more readable to add the one flag we need in the  two `spawnPromisified` calls we have
```suggestion
const execOptions = '--input-type=module';
```

Reply:
if I commit this change it will break the test as you haven't adjusted the string unpacking further on. also, if we desire to add other flags here, it will have to be changed back to an array. I am not convinced.

Reply:
What other flag though?

Reply:
Until that future happens, could we please simplify that?

Reply:
I'll inline the flag. I'd like to make it known I am unimpressed 😂